src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -177,9 +177,19 @@ <h1 class="subheader__title" property="name">{% render_model current_page "title
                     {% block run_open_single %}
                         {% with runs_dict=course.course_runs_dict %}
                             {% with open_visible_runs=runs_dict.0|add:runs_dict.1|add:runs_dict.2|visible_on_course_page:request.toolbar.edit_mode_active %}
+                            <!-- TODO(rlecellier): remove this before merge -->
+                                <div class="course-detail__row course-detail__runs course-detail__runs--open">
+                                    <div class="course-detail__empty">{% trans "No open course runs" %}</div>
+                                    <div
+                                        class="richie-react richie-react--course-add-to-wishlist"
+                                        data-props='{"courseCode":"{{ course.code }}"}'></div>
+                                </div>
                                 {% if open_visible_runs|length == 0 %}
                                     <div class="course-detail__row course-detail__runs course-detail__runs--open">
                                         <div class="course-detail__empty">{% trans "No open course runs" %}</div>
+                                        <div
+                                            class="richie-react richie-react--course-add-to-wishlist"
+                                            data-props='{"courseCode":"{{ course.code }}"}'></div>
```

Comment:
Should be display only if Joanie is enabled.
You can use the template tag `is_joanie_enabled`

https://github.com/openfun/richie/blob/master/src/richie/apps/core/templatetags/joanie.py#L9-L16

`{% if is_joanie_enabled %}...{% endif %}`